src/synth/fluid_voice.c

```diff
@@ -488,22 +488,7 @@ fluid_voice_calculate_gen_pitch(fluid_voice_t* voice)
 
 }
 
-/*
- * fluid_voice_calculate_runtime_synthesis_parameters
- *
- * in this function we calculate the values of all the parameters. the
- * parameters are converted to their most useful unit for the DSP
- * algorithm, for example, number of samples instead of
- * timecents. Some parameters keep their "perceptual" unit and
- * conversion will be done in the DSP function. This is the case, for
- * example, for the pitch since it is modulated by the controllers in
- * cents. */
-static int
-fluid_voice_calculate_runtime_synthesis_parameters(fluid_voice_t* voice)
-{
-  int i;
-
-  int list_of_generators_to_initialize[35] = {
+static int const list_of_generators_to_initialize[] = {
```

Comment:
Really move this to global scope? It only used by `fluid_voice_calculate_runtime_synthesis_parameters()` so IMO it should only be visible in that function.